OpenRA.Game/FileSystem/InstallShieldCABExtractor.cs

```diff
@@ -404,6 +404,16 @@ public string DirectoryName(uint index)
 			return test;
 		}
 
+		public uint GetIndexFromFilename(string directory, string filename)
+		{
+			var path = directory + "\\" + filename;
```

Comment:
https://gist.github.com/DSUK/e243103217bac918b175 yes they are inside... you say in this case give me the IDs of the files... not give me the files of the IDs.